stdlib/Profile/src/Profile.jl

```diff
@@ -1196,14 +1196,14 @@ function rtruncto(str::String, w::Int)
     if length(str) <= w
         return str
     else
-        return string("...", str[prevind(str, end, w-4):end])
+        return string("…", str[prevind(str, end, w-2):end])
     end
 end
 function ltruncto(str::String, w::Int)
     if length(str) <= w
```

Comment:
Can you fix this to use textwidth as well then, especially since this is now using unicode